indico/modules/groups/controllers.py

```diff
@@ -31,7 +32,15 @@
 from indico.web.rh import RHProtected
 
 
-class RHGroups(RHAdminBase):
+class RHLocalGroupConfig:
```

Comment:
All classes in Python 2 should be new-style, i.e. inherit from `object` (or another base class that already inherits from `object`).

Since this particular one is a mixin, it should be named e.g. `LocalGroupConfigMixin` - but I don't think a mixin is the best choice here, since you can just have this class (renamed to `RHLocalGroupsBase`) inherit from `RHAdminBase`, add the check here, and then inherit all the other classes that currently use `RHAdminBase` directly from `RHLocalGroupsBase` instead.